.golangci.yml

```diff
@@ -1,11 +1,61 @@
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
+  disable-all: true
+  enable:
+    - asciicheck
+    - bodyclose
+    - deadcode
+    - depguard
+    # - dogsled (TODO: write rational for disabling here, brought over from Makefile)
+    # - dupl (TODO: write rational for disabling here, brought over from Makefile)
+    - errcheck
+    - exhaustive
+    - funlen
+    # - gochecknoglobals (TODO: write rational for disabling here, brought over from Makefile)
+    # - gochecknoinits (TODO: write rational for disabling here, brought over from Makefile)
+    - gocognit
+    # - goconst (TODO: write rational for disabling here, brought over from Makefile)
+    - gocritic
+    - gocyclo
+    - godot
+    - godox
+    # - goerr113 (TODO: expose errors proerply, once/if approaching library use, as oposed to current cli-only use)
+    - gofmt
+    - goimports
+    - golint
+    # - gomnd (TODO: write rational for disabling here, brought over from Makefile)
+    - goprintffuncname
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    # - lll (TODO: write rational for disabling here, brought over from Makefile)
```

Comment:
Following `gofmt` principles, we aren't enforcing a line length at this point. Most code is well formatted, long lines happen only in exceptional cases like when code contains long URLs. The long line is acceptable in those situations.